lib/ssl/src/ssl.erl

```diff
@@ -453,130 +584,90 @@ only the [signature_algs](`t:signature_algs/0`) extension is sent.
--doc """
-The user's private key. Either the key can be provided directly as DER encoded
-entity, or indirectly using a crypto engine/provider (with key reference
-information) or an Erlang fun (with possible custom options). The latter two
-options can both be used for customized signing with for instance hardware
-security modules (HSM) or trusted platform modules (TPM).
 
-- A DER encoded key will need to specify the ASN-1 type used to create the
-  encoding.
-- An engine/provider needs to specify specific information to support this
-  concept and can optionally be password protected, see also
-  [crypto:engine_load/3 ](`crypto:engine_load/3`)and
-  [Crypto's Users Guide](`e:crypto:engine_load.md`).
-- A fun option should include a fun that mimics `public_key:sign/4` and possibly
-  [public_key:private_encrypt/4](`public_key:encrypt_private/3`) if legacy
-  versions TLS-1.0 and TLS-1.1 should be supported.
 
-If this option is supplied, it overrides option `keyfile`.
-""".
--doc(#{title =>
-           <<"TLS/DTLS OPTION DESCRIPTIONS - COMMON for SERVER and CLIENT">>}).
--type key()                       :: {'RSAPrivateKey'| 'DSAPrivateKey' | 'ECPrivateKey' |'PrivateKeyInfo', 
-                                      public_key:der_encoded()} |
-                                     #{algorithm := sign_algo(),
-                                       engine := crypto:engine_ref(), 
-                                       key_id := crypto:key_id(), 
-                                       password => crypto:password()} |
-                                     #{algorithm := sign_algo(),
-                                       sign_fun := fun(),
-                                       sign_opts => list(),
-                                       encrypt_fun => fun(), %% Only TLS-1.0, TLS-1.1 and rsa-key
-                                       encrypt_opts => list()
-                                      }. % exported
--doc """
-Path to the file containing the user's private PEM-encoded key. As PEM-files can
-contain several entries, this option defaults to the same file as given by
-option `certfile`.
-""".
--doc(#{title =>
-           <<"TLS/DTLS OPTION DESCRIPTIONS - COMMON for SERVER and CLIENT">>}).
--type key_pem()                   :: file:filename().
--doc """
-String containing the user's password or a function returning same type. Only
-used if the private keyfile is password-protected.
+- **\{keep_secrets, KeepSecrets}** -
+Configures a TLS 1.3 connection for keylogging
+
+In order to retrieve keylog information on a TLS 1.3 connection, it must be
+configured in advance to keep the client_random and various handshake secrets.
+
+The keep_secrets functionality is disabled (`false`) by default.
+
+Added in OTP 23.2
+
+- **\{handshake_size, HandshakeSize}** -
+Integer (24 bits unsigned). Used to limit the size of valid TLS handshake
+packets to avoid DoS attacks. Defaults to 256\*1024.
+
+- **\{hibernate_after, HibernateTimeout}** -
```

Comment:
```suggestion
- **{hibernate_after, HibernateTimeout}** -
```

You can remove this `\` if you want to. It was inserted by an overzelous escape algorithm. There are many places where this mistake is done, I've only commented on this one.